<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMManagerMainWindow</class>
 <widget class="QMainWindow" name="VMManagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionCheck_for_updates"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Machine"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartPause"/>
   <addaction name="actionHard_Reset"/>
   <addaction name="actionForce_Shutdown"/>
   <addaction name="actionCtrl_Alt_Del"/>
   <addaction name="actionSettings"/>
   <addaction name="actionNew_Machine"/>
  </widget>
  <action name="actionDo_something">
   <property name="text">
    <string>Do something</string>
   </property>
  </action>
  <action name="actionStartPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/menuicons/qt/icons/run.ico</normaloff>:/menuicons/qt/icons/run.ico</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHard_Reset">
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/menuicons/qt/icons/hard_reset.ico</normaloff>:/menuicons/qt/icons/hard_reset.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Hard Reset...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionForce_Shutdown">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/menuicons/qt/icons/acpi_shutdown.ico</normaloff>:/menuicons/qt/icons/acpi_shutdown.ico</iconset>
   </property>
   <property name="text">
    <string>Force shutdown</string>
   </property>
   <property name="toolTip">
    <string>Force shutdown</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCtrl_Alt_Del">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/menuicons/qt/icons/send_cad.ico</normaloff>:/menuicons/qt/icons/send_cad.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Ctrl+Alt+Del</string>
   </property>
   <property name="toolTip">
    <string>Ctrl+Alt+Del</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/menuicons/qt/icons/settings.ico</normaloff>:/menuicons/qt/icons/settings.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionNew_Machine">
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/settings/qt/icons/86Box-yellow.ico</normaloff>:/settings/qt/icons/86Box-yellow.ico</iconset>
   </property>
   <property name="text">
    <string>New Machine</string>
   </property>
   <property name="toolTip">
    <string>New Machine</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qt_resources.qrc">
     <normaloff>:/menuicons/qt/icons/run.ico</normaloff>:/menuicons/qt/icons/run.ico</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
